src/Akeneo/Platform/Job/front/process-tracker/src/feature/models/StepExecutionRowTracking.test.ts

```diff
@@ -19,6 +19,7 @@ test('it can get the badge level for a given step tracking', () => {
   expect(getStepExecutionRowTrackingLevel({...stepTracking, has_error: true})).toEqual('danger');
   expect(getStepExecutionRowTrackingLevel({...stepTracking, warning_count: 2})).toEqual('warning');
   expect(getStepExecutionRowTrackingLevel({...stepTracking, has_error: true, warning_count: 2})).toEqual('danger');
+  expect(getStepExecutionRowTrackingLevel({...stepTracking, status: 'PAUSED'})).toEqual('tertiary');
```

Comment:
If it's a requirement
```suggestion
  expect(getStepExecutionRowTrackingLevel({...stepTracking, status: 'PAUSED'})).toEqual('tertiary');
  expect(getStepExecutionRowTrackingLevel({...stepTracking, status: 'PAUSED', has_error: true})).toEqual('danger');
  expect(getStepExecutionRowTrackingLevel({...stepTracking, status: 'PAUSED', warning_count: 2})).toEqual('warning');
```